apps/web/pages/apps/index.tsx

```diff
@@ -52,7 +52,11 @@ export default function Apps({
           <TrendingAppsSlider items={appStore} />
         </>
       )}
-      <AllApps apps={appStore} searchText={searchText} />
+      <AllApps
+        apps={appStore}
+        searchText={searchText}
+        categories={categories.map((category: typeof categories[number]) => category.name)}
```

Comment:
```suggestion
        categories={categories.map((category) => category.name)}
```
If inferred types are failing let's fix that instead